### PR TITLE
Initial implementation of tapered extrusion

### PR DESCRIPTION
**Pull request by adam-urbanczyk**
Tapred `extrude` and `cutBlind` operations. Teaser pic: 
afbeelding

Related to #70

### CODE REVIEW

**Comment by dcowden** [comment]
cool addition Adam!

**Comment by adam-urbanczyk** [comment]
I think it is ready to be merged. Somehow `cutBlind` requires negative depths. I wonder if it would make sense to reverse this sign convention.

NB: the coverage misses are not real - I did not touch the red files.

File: .travis.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ before_install:
 - export PATH="$HOME/miniconda/bin:$HOME/miniconda/lib:$PATH";
 - hash -r;
 - conda config --set always_yes yes --set changeps1 no;
-- conda create -y -q -n test_cq -c pythonocc -c oce -c conda-forge -c dlr-sc pythonocc-core=0.18
+- conda create -y -q -n test_cq -c pythonocc -c oce -c conda-forge -c dlr-sc pythonocc-core=0.18.1
```
Comment (jmwright) [inline .travis.yml]: The .travis.yml file on the adam-urbanczyk-OCC-version-update branch pulls pythonocc-core 0.18.2. Should we standardize on a version, or does it matter?

**Comment by jmwright** [comment]
> Somehow `cutBlind` requires negative depths.

I've run into this periodically and just invert the normal on the workplane. I think the normal is oriented to assume extrusion, and so a cut operation requires a negative depth unless the normal is inverted. At least I think that's what's happening.

> NB: the coverage misses are not real - I did not touch the red files.

I have annoyances like this with coverage in this project and others. There's probably a way to make coverage less sensitive, but I have never checked on that option.

**Comment by dcowden** [comment]
It looks good to me.

File: .travis.yml (hunk 1 above)
Comment (adam-urbanczyk) [reply .travis.yml]: Yes that is on purpose. I wan't to keep 0.18.1 explicitly for now (that is what master used to be tested with). Once the issues with 0.18.2 (Mac) are solved we can switch to that version.

**Comment by adam-urbanczyk** [comment]
OK, merging. Tagging @osterwood for info.

**Pull request merged**